python/ray/tune/trainable.py

```diff
@@ -174,11 +175,11 @@ class Trainable:
     Calling ``save()`` should save the training state of a trainable to disk,
     and ``restore(path)`` should restore a trainable to the given state.
 
-    Generally you only need to implement ``_setup``, ``_train``,
-    ``_save``, and ``_restore`` when subclassing Trainable.
+    Generally you only need to implement ``build``, ``step``,
+    ``save_checkpoint``, and ``load_checkpoint`` when subclassing Trainable.
```

Comment:
Somewhat unrelated to this PR, but it might be worth mentioning here that the minimal class-based setup only requires `build` and `step`.

```suggestion
    Minimal `Trainable` needs to implement ``build`` and ``step`` methods.
    Additionally, ``save_checkpoint``, and ``load_checkpoint`` enable saving
    and restoring models.
```